packages/api/test/e2e/api-derive/promise-dev.spec.ts

```diff
@@ -73,7 +73,7 @@ describeE2E({
         }
 
         expect(info.accountId.eq(accountId)).toBe(true);
-        expect(info.controllerId.eq('5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY')).toBe(true);
+        expect(info.controllerId!.eq('5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY')).toBe(true);
```

Comment:
A side note, we need to not add any more of these, i.e. https://github.com/polkadot-js/api/blob/master/.eslintrc.js#L10

This is the TypeScript equivalent of raw `unwrap()` without a check in Rust. It is passable in tests, however we need to remove that rule override since it is a code smell. (Disallowed by default by the standard rules, e.g. the recommended set)